spec/formatting.md

```diff
@@ -77,50 +77,96 @@ At a minimum, it includes:
   This is often determined by a user-provided argument of a formatting function call.
 
 - The _function registry_,
-  providing the implementations of the functions referred to by message _functions_.
+  providing the _function handlers_ of the functions referred to by message _functions_.
 
 - Optionally, a fallback string to use for the message if it is not _valid_.
 
 Implementations MAY include additional fields in their _formatting context_.
 
-## Expression and Markup Resolution
+## Resolved Values
 
-_Expressions_ are used in _declarations_ and _patterns_.
-_Markup_ is only used in _patterns_.
+This specification allows for the same value to be used for:
+- formatting in a _placeholder_,
+- selection with a _selector_,
+- as the _operand_ of another _expression_ (including _local declarations_), or
+- as an _option_ value in another _expression_,
 
-In a _declaration_, the resolved value of the _expression_ is bound to a _variable_,
+To support this, the **_<dfn>resolved value</dfn>_** of each _expression_
```

Comment:
If we're going to `dfn` _resolved value_ here, we should write a self contained, concise definition, since every occurrence of the term will come here. On a mobile so not suggesting text just now.